ui/pages/confirmations/components/confirm-page-container/index.scss

```diff
@@ -2,9 +2,6 @@
 @import 'confirm-page-container-header/index';
 @import 'confirm-detail-row/index';
 @import 'confirm-page-container-navigation/index';
-///: BEGIN:ONLY_INCLUDE_IF(snaps)
```

Comment:
why was this removed?

Reply:
@hmalik88 : `confirm-page-container` has now been moved to `pages/confirmations` as they are ownership of confirmations team and so I moved `snap-insight` to `/components/app/spans`. This is the reason for above change and also change in file [ui/components/app/app-components.scss](https://github.com/MetaMask/metamask-extension/pull/22639/files/6f6b9cf4936eaaed2d5190b40582109a9600b3c1#diff-210a3b6ffa98ce55e09288b37de7aa678ecf0e75015bc42786eee28f940552ba)

Reply:
Gotcha, thanks!